### PR TITLE
Change ``tab_width``->``indentation_width``

### PR DESCRIPTION
**Pull request by Asnelchristian**

### CODE REVIEW

**Comment by abhsag24** [comment]
why do i see 2 commits? :P did you not ammend the previous commit? or is it a github UI issue?

File: bears/coffee_script/CoffeeLintBear.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def generate_config(filename, file,
             Checks if the file ends with a single newline.
         :param use_spaces:
             Forbids tabs in indentation and applies two spaces for this purpose.
-        :param tab_width:
+        :param indentation_width:
             Length of the tab for indentation.
```
Comment (abhsag24) [inline bears/coffee_script/CoffeeLintBear.py]: probably might want to change this as well

File: bears/coffee_script/CoffeeLintBear.py [hunk 2]
```diff
@@ -98,8 +98,9 @@ def generate_config(filename, file,
             Checks if the file ends with a single newline.
         :param use_spaces:
             Forbids tabs in indentation and applies two spaces for this purpose.
-        :param tab_width:
-            Length of the tab for indentation.
+        :param indentation_width:
+            No. of spaces to insert for indentation.
+            Only Applicable if use_spaces is False.
```
Comment (sils1297) [inline bears/coffee_script/CoffeeLintBear.py]: you also need it to calculate how tabs should be visualized

**Comment by sils1297** [comment]
can yo uget the tests passing?

**Comment by sils1297** [comment]
marking WIP in the meantime

**Comment by Asnelchristian** [comment]
i really don't know why this is failing ;(
Moving it to https://github.com/coala-analyzer/coala-bears/pull/684

**Pull request closed without merging**